presets/4.3/mixed/karate race tune preset.txt

```diff
@@ -0,0 +1,102 @@
+# Title: Karate Race
+# FirmwareVersion: 4.3
+# Category: TUNE
+# Official: true
+# Keywords: karate, 6S, race, world champion, 5 inch, 5"
+# Author: Zak Smiley SugarK
+# Description: This 6s racing tune was developed with the help of @ctzsnooze around @karatebrot’s RPM crossfade code allowing a pretty aggressive filter array which is the karate array. This tune works well on any quality airframes and has been flown on the Viper racer (where it was developed ) BMS Js1, 533 switchback and the HGLRC wind. It should also work on other quality racing frames but use at your own risk as it is an aggressive tune. This tune so far has won a good number of events including the 2021 IO.
+# Description: Things to note.. YOU have to use RPM filtering on this tune, failure to do so might result in fire and regarding Dshot600,  if your setup has errors in the motor tab using bidirectional Dshot then change to 8k/4k and dshot300. Also this should be applied to a clean flash after you've setup your rates, swtiches, vtx tables etc. To test arm the quad with props on, it should sound clean with no grinding, if it passes that then hover test and check motor temps. If any thing is off don't fly it!!
+# Description: Radio links.. 1. Make sure your radio system is totally up to date using either Edgetx or Opentx and your ADC in the hardware page is OFF 2. Go to the radio presets and apply the correct setup for your system and link speed.
+
+# include: presets/4.3/tune/defaults.txt
+
+# master
+set gyro_lpf2_static_hz = 1000
+set dyn_notch_count = 2
+set dyn_notch_q = 500
+set dyn_notch_min_hz = 200
+set gyro_lpf1_dyn_min_hz = 500
+set gyro_lpf1_dyn_max_hz = 1000
+set dshot_bidir = ON
+set motor_pwm_protocol = DSHOT600
+set small_angle = 180
+set simplified_gyro_filter_multiplier = 200
+set debug_mode = GYRO_SCALED
+set rpm_filter_fade_range_hz = 100
+set name = karate race
+
+
+# pidprofile
+set dterm_lpf1_dyn_expo = 10
+set anti_gravity_gain = 5500
+set p_pitch = 42
+set i_pitch = 89
+set d_pitch = 41
+set f_pitch = 138
+set p_roll = 38
+set i_roll = 81
+set d_roll = 37
+set f_roll = 125
+set d_min_roll = 22
+set d_min_pitch = 24
+set feedforward_jitter_reduction = 5
```

Comment:
Sorry for being picky.  feedforward_jitter_reduction is a part of RC_LINK or RC_SMOOTHING I believe? Not the TUNE. Again, @ctzsnooze  :)